Makefile

```diff
@@ -11,7 +11,8 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and
 # limitations under the License.
-VERSION := $(shell git describe --tags | sed -e 's/v//' -e 's/-.*//')
+ROOT := $(shell pwd)
+VERSION := $(shell cat $(ROOT)/ecs-init/ECSVERSION)
```

Comment:
setting and using `ROOT` in this way is redundant. Just do:
`VERSION := $(shell cat ecs-init/ECSVERSION)`

You probably also want `=` instead of `:=`

Reply:
yeah, updated